src/server/command/worldedit/paste.zig

```diff
@@ -19,9 +19,10 @@ pub fn execute(args: []const u8, source: *User) void {
 		return;
 	}
 	source.mutex.lock();
-	defer source.mutex.unlock();
+	const nullableClipboard = source.commandData.clipboard;
+	source.mutex.unlock();
```

Comment:
Another thread may deinit the clipboard while we still hold a reference to it. The lock is too tight here.